app/src/ui/app.tsx

```diff
@@ -583,6 +585,23 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClickDarwin() {
+    const actionOnDoubleClick: AppleActionOnDoubleClickPref = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
```

Comment:
It should already be on since we have [strict](https://www.typescriptlang.org/docs/handbook/compiler-options.html) enabled: https://github.com/desktop/desktop/blob/6a14dfa0318ff81b5fdc9cc00e15a2fab19878e6/tsconfig.json#L11